packages/react-instantsearch-hooks/src/__tests__/useCurrentRefinements.test.tsx

```diff
@@ -0,0 +1,96 @@
+import { renderHook } from '@testing-library/react-hooks';
+import React from 'react';
+
+import { RefinementList } from '../../../../examples/hooks/components';
```

Comment:
We shouldn't use a component from the examples in our lib tests. This is sufficient:

```tsx
function RefinementList(props: UseRefinementListProps) {
  useRefinementList(props);
  return null;
}
```

Reply:
Yep I saw that, I'm already on it ;)